src/main/java/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftBuildVerifier.java

```diff
@@ -70,64 +72,107 @@ public String getWaitTime() {
-            return "openshiftVerifyBuild";
-        }
-
-        @Override
-        public String getDisplayName() {
-            return DISPLAY_NAME;
-        }
-
-        @Override
-        public Step newInstance(Map<String, Object> arguments) throws Exception {
-            if (!arguments.containsKey("buildConfig") && !arguments.containsKey("bldCfg"))
-            	throw new IllegalArgumentException("need to specify buildConfig");
-            Object bldCfg = arguments.get("buildConfig");
-            if (bldCfg == null || bldCfg.toString().length() == 0)
-            	bldCfg = arguments.get("bldCfg");
-            if (bldCfg == null || bldCfg.toString().length() == 0)
-            	throw new IllegalArgumentException("need to specify buildConfig");
-            OpenShiftBuildVerifier step = new OpenShiftBuildVerifier(bldCfg.toString());
-            if (arguments.containsKey("checkForTriggeredDeployments")) {
-                Object checkForTriggeredDeployments = arguments.get("checkForTriggeredDeployments");
-                if (checkForTriggeredDeployments != null) {
-                    step.setCheckForTriggeredDeployments(checkForTriggeredDeployments.toString());
-                }
-            }
-            if (arguments.containsKey("waitTime")) {
-                Object waitTime = arguments.get("waitTime");
-                if (waitTime != null) {
-                    step.setWaitTime(waitTime.toString());
-                }
-            }
-            ParamVerify.updateDSLBaseStep(arguments, step);
-            return step;
-        }
-    }
+	public static class DescriptorImpl extends AbstractStepDescriptorImpl implements IOpenShiftTimedStepDescriptor {
+		private String waitUnit = "sec";
+		private String wait;
+
+		public DescriptorImpl() {
+			super(OpenShiftBuildVerifierExecution.class);
+		}
+
+		@Override
+		public String getFunctionName() {
+			return "openshiftVerifyBuild";
+		}
+
+		@Override
+		public String getDisplayName() {
+			return DISPLAY_NAME;
+		}
+
+		@Override
+		public Step newInstance(Map<String, Object> arguments) throws Exception {
+			if (!arguments.containsKey("buildConfig") && !arguments.containsKey("bldCfg"))
+				throw new IllegalArgumentException("need to specify buildConfig");
+			Object bldCfg = arguments.get("buildConfig");
+			if (bldCfg == null || bldCfg.toString().length() == 0)
+				bldCfg = arguments.get("bldCfg");
+			if (bldCfg == null || bldCfg.toString().length() == 0)
+				throw new IllegalArgumentException("need to specify buildConfig");
+			OpenShiftBuildVerifier step = new OpenShiftBuildVerifier(bldCfg.toString());
+			if (arguments.containsKey("checkForTriggeredDeployments")) {
+				Object checkForTriggeredDeployments = arguments.get("checkForTriggeredDeployments");
+				if (checkForTriggeredDeployments != null) {
+					step.setCheckForTriggeredDeployments(checkForTriggeredDeployments.toString());
+				}
+			}
+
+			if (arguments.containsKey("waitTime")) {
```

Comment:
the `doFillWaitArguments` call I see in other step's `newInstance` is missing here ... was that intentional?